crates/sui-indexer-alt-framework/src/pipeline/concurrent/commit_watermark.rs

```diff
@@ -27,37 +27,37 @@ use super::Handler;
 /// updating its row in the `watermarks` table when a continuous run of checkpoints have landed
 /// since the last watermark update.
 ///
-/// It receives watermark "parts" that detail the proportion of each checkpoint's data that has
-/// been written out by the committer and periodically (on a configurable interval) checks if the
+/// It receives watermark "parts" that detail the proportion of each checkpoint's data that has been
+/// written out by the committer and periodically (on a configurable interval) checks if the
 /// watermark for the pipeline can be pushed forward. The watermark can be pushed forward if there
 /// is one or more complete (all data for that checkpoint written out) watermarks spanning
 /// contiguously from the current high watermark into the future.
 ///
-/// If it detects that more than [WARN_PENDING_WATERMARKS] watermarks have built up, it will issue
-/// a warning, as this could be the indication of a memory leak, and the caller probably intended
-/// to run the indexer with watermarking disabled (e.g. if they are running a backfill).
+/// If it detects that more than [WARN_PENDING_WATERMARKS] watermarks have built up, it will issue a
+/// warning, as this could be the indication of a memory leak, and the caller probably intended to
+/// run the indexer with watermarking disabled (e.g. if they are running a backfill).
 ///
 /// The task regularly traces its progress, outputting at a higher log level every
 /// [LOUD_WATERMARK_UPDATE_INTERVAL]-many checkpoints.
 ///
-/// The task will shutdown if the `cancel` token is signalled, or if the `rx` channel closes and
-/// the watermark cannot be progressed. If `skip_watermark` is set, the task will shutdown
-/// immediately.
+/// The `watermark_key` is used to set and read the committer watermark. This will be the same value
```

Comment:
nit: docs need updating -- no more `watermark_key` here, refer to variable names rather than flags for doc comments in internal functions.